bin/deploy

```diff
@@ -1,6 +1,10 @@
 #! /usr/bin/env bash
 
-# DOC: Deploy an image. Usage: bin/deploy --tag=<image tag> --config=<config_file>
+# DOC: Deploy an image. Usage: bin/deploy --tag=<image tag> 
+# Optional params:
+#   --config=<config_file> to override config file
+#   --infra_commit=<commit_sha> to override infrastructure version. 
+#     Needs to be a valid config from https://github.com/civiform/civiform.
```

Comment:
Valid config or valid commit? Here and below.

Reply:
good catch